website/www/site/i18n/home/calendar/en.yaml

```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
 
 - id: home-calendar-title
-  translation: "Stay up to date with Beam"
+  translation: "Stay Up To Date with Beam"
```

Comment:
I'd stick with sentence case instead of title case, but feel free to ignore...

Reply:
Thank you, @pcoet! As discussed on today's call to keep title case everywhere.